src/components/world-map/world-map.tsx

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from 'react';
+import { MapContainer, TileLayer, CircleMarker, ZoomControl } from 'react-leaflet';
+
+import { API_ROUTES } from 'src/constants';
+import { Buoy, ParsedBuoy } from 'src/types';
+import './world-map.css';
+
+export const WorldMap = () => {
+  const [buoys, setBuoys] = useState<ParsedBuoy[]>([]);
+
+  const parseBuoyData = (data: Buoy[]) => {
+    const parsedBuoyObjArr = [];
+    for (const buoy of data) {
+      const { station_id, name, location } = buoy;
+      const newBuoyObj = {
+        latitude: location[1],
+        longitude: location[0],
+        name,
```

Comment:
Why is name standing out here all alone? Why is it a part of longitude?

Reply:
It's not part of longitude due to comma separation. When a variable is referenced in an object without an explicit definition for its key in JS/TS, the variable name becomes the name of key by default.

What I wrote is equivalent to the following:
```ts
const newBuoyObj = {
    latitude: location[1],
    longitude: location[0],
    name: name,
}
```

Reply:
Got it!